### PR TITLE
Deprecate shared worker

### LINKED ISSUE #115: Auth Lobby fails to complete initialization in Safari 15.5
Browser: Safari 15.5 (15613.2.7.1.9, 15613)
OS: macOS 10.15.7 (19H1922)

Ok, I'm having trouble isolating whether this a single issue or multiple issue. I'm going to use this issue to catalogue all the problems for now, and we can create new issues as needed. It _seems_ like all of these issues are related to the fact that we load `js-ipfs` in an iframe, so we're getting nailed by privacy and cross-origin features.

## Mumblr

- Application URL: https://mumblr.app

Hangs on initialize. Fails silently, likely due to inadequate error checking.

## Contacts

- Application URL: https://contacts.fission.app

Hangs on initialize. Fails silently, likely due to inadequate error checking.

## Dashboard

- Application URL: https://dashboard.fission.codes

Error: hangs during initialization
```
[Error] Error in webnative initialisation
TypeError: undefined is not an object (evaluating 'a.metadata.cid')
(anonymous function) — index.ts:472
asyncFunctionResume
(anonymous function)
promiseReactionJobWithoutPromise
promiseReactionJob
	(anonymous function) (bundle.min.js:42:462730)
	promiseReactionJob
```

## Drive

- Application URL: https://drive.fission.codes

Error: hangs during intialization
```
[Error] TypeError: undefined is not an object (evaluating 'a.metadata.cid')
(anonymous function) — index.ts:97
asyncFunctionResume
(anonymous function)
promiseReactionJobWithoutPromise
promiseReactionJob

	(anonymous function) (index.js:148)
	promiseReactionJob
```
and 
```
[Error] Unhandled Promise Rejection: Error: https://github.com/elm/core/blob/1.0.0/hints/4.md
	_Debug_crash (application.js:460:88)
	send (application.js:2298)
	(anonymous function) (index.js:153)
	promiseReactionJob
```

### CBOR Error

At a different point, I saw a `CBOR Error`, that I can't reproduce at the moment. I did save a screenshot:
Screen Shot 2022-07-12 at 4 10 05 PM

```
[Error] Error: CBOR decode error: did not find any content to decode
Lm — index.ts:97
asyncFunctionResume
(anonymous function)
promiseReactionJobWithoutPromise
promiseReactionJob

	(anonymous function) (index.js:148)
	promiseReactionJob
```
and
```
[Error] Unhandled Promise Rejection: Error: https://github.com/elm/core/blob/1.0.0/hints/4.md
	_Debug_crash (application.js:460:88)
	send (application.js:2298)
	(anonymous function) (index.js:153)
	promiseReactionJob
```

### PR DESCRIPTION
**Pull request by icidasset**
Switches to the new webnative that uses a new method to connect to IPFS (ie. no more shared worker). We'll still keep the shared worker around for now for backwards compatibility.

Closes #115

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ workbox_config := "workbox.config.cjs"
 # -----
 
 @default: dev-build
-	just dev-server & just watch
+	just dev-server # & just watch
```
Comment (matheus23) [inline Justfile:19]: Was this change intentional?

**Review by matheus23 — APPROVED** [review]
:tada:

File: Justfile (hunk 1 above)
Comment (icidasset) [reply Justfile:19]: Ugh kind of, disabled watching because of a Nix issue `watchexec` is no longer able to compile on ARM Macs (something something Apple SDK also causing issues with `cargo watch`)

Comment (icidasset) [reply Justfile:19]: I'll revert the change.
